### ISSUE TITLE
Running notebooks in Colab and pip installation on Mac OS

### ISSUE THREAD

**Issue by Slowika** [microsoft/TextWorld#209]
I'm having issues with installing TextWorld in Colab and Mac OS (Mojave). By following the basic setup instructions, I have only managed to install it on a Linux machine.

The error in Colab when installing TextWorld with pip: 

```
ERROR: stable-baselines 2.2.1 has requirement gym[atari,classic_control]>=0.10.9, but you'll have gym 0.10.4 which is incompatible.
ERROR: jupyter-console 5.2.0 has requirement prompt-toolkit<2.0.0,>=1.0.0, but you'll have prompt-toolkit 2.0.10 which is incompatible.
ERROR: ipython 5.5.0 has requirement prompt-toolkit<2.0.0,>=1.0.4, but you'll have prompt-toolkit 2.0.10 which is incompatible.
ERROR: dopamine-rl 1.0.5 has requirement gym>=0.10.5, but you'll have gym 0.10.4 which is incompatible.
```

On Mac `pip install textworld` is still experiencing the issue https://github.com/microsoft/TextWorld/issues/170. Cloning the source code and running `pip install .` works so the pip version of TextWorld might be broken on Mac.

**Comment by MarcCote** [comment]
@Slowika thanks for reporting this. You are correct the master branch contains all the recent fixes and upgrades. I'm currently working on a release to update the `textworld` library on pypi.

**Comment by MarcCote** [comment]
Closed by #213
